### PR TITLE
select and render ada accomodation in client profile

### PR DESCRIPTION
**Pull request by tglaz**
### FE - Client profile ADA Accommodation
https://betterangels.atlassian.net/browse/DEV-708

* Mainly I'm curious if didn't do anything silly with the gql setup and did code generation correctly etc. But there's a comment below about `purpose` becoming optional
* For the most part I tried to follow the patterns that already exist. There's some opportunity to reduce a bit of the boilerplate, but can discuss another time.

### CODE REVIEW

File: libs/expo/betterangels/src/lib/apollo/graphql/__generated__/types.ts [hunk 1]
```diff
@@ -289,7 +289,7 @@ export type CreateNoteInput = {
   client?: InputMaybe<Scalars['ID']['input']>;
   privateDetails?: InputMaybe<Scalars['String']['input']>;
   publicDetails?: InputMaybe<Scalars['String']['input']>;
-  purpose: Scalars['String']['input'];
+  purpose?: InputMaybe<Scalars['String']['input']>;
```
Comment (tglaz) [inline libs/expo/betterangels/src/lib/apollo/graphql/__generated__/types.ts:292]: @mikefeldberg  - would you know if this is related to your comments in PR #607 about purpose becoming optional?
This ended up in my PR although it probably shouldn't have and is failing linting jobs. Or did I do something weird with my gql type generation?

**Comment by tglaz** [comment]
I noticed that the `reset` button actually just clears the forms, as opposed to setting the existing value. Is this intended or a bug?

**Comment by tglaz** [comment]
and tests pass : )_

image

**Review by mikefeldberg — APPROVED** [review]

File: libs/expo/betterangels/src/lib/screens/AddEditClient/AddEditClient.graphql [hunk 2]
```diff
@@ -59,6 +59,7 @@ query GetClientProfile($id: ID!) {
       displayPronouns
```
Comment (shiozakiyuko) [inline libs/expo/betterangels/src/lib/screens/AddEditClient/AddEditClient.graphql:59]: Hi @tom, It looks good to me!

**Review by shiozakiyuko — APPROVED** [review]

**Pull request merged**